mkdocs.yml

```diff
@@ -75,6 +75,7 @@ nav:
         - Custom Fields: 'additional-features/custom-fields.md'
         - Custom Links: 'models/extras/customlink.md'
         - Export Templates: 'models/extras/exporttemplate.md'
+        - Filters: 'additional-features/filters.md'
```

Comment:
Wonder if we should elaborate on this as `Template Filters` or similar? To make it less ambiguous with concepts like queryset filters?

Reply:
Yes, I think we should.